webots/controllers/falling/falling.py

```diff
@@ -0,0 +1,40 @@
+from controller import Supervisor
+from simple_websocket_server import WebSocketServer, WebSocket
```

Comment:
This package is not part of the standard library nor is it provided by webots automagically.
Please add it to the `requirements.txt` file.

Reply:
Removed unnecessary imports and unused references